Online/Menus/CSS/LoadCSSText.asm

```diff
@@ -1170,6 +1170,7 @@ CSS_ONLINE_CHAT_THINK:
 blrl
 mr REG_CHATMSG_GOBJ, r3 # Store GOBJ pointer
 backup
+fbackup 2
```

Comment:
`fbackup` and `frestore` are replacements for the normal `backup` and `restore`. You shouldn't be using both.